Formula/ffmpeg@5.1.rb

```diff
@@ -0,0 +1,140 @@
+class FfmpegAT51 < Formula
+  desc "Play, record, convert, and stream audio and video"
+  homepage "https://ffmpeg.org/"
+  url "https://ffmpeg.org/releases/ffmpeg-5.1.3.tar.xz"
+  sha256 "1b113593ff907293be7aed95acdda5e785dd73616d7d4ec90a0f6adbc5a0312e"
+  # None of these parts are used by default, you have to explicitly pass `--enable-gpl`
+  # to configure to activate them. In this case, FFmpeg's license changes to GPL v2+.
+  license "GPL-2.0-or-later"
+
+  livecheck do
+    url "https://ffmpeg.org/download.html"
+    regex(/href=.*?ffmpeg[._-]v?(5\.1(?:\.\d+)+)\.t/i)
+  end
+
+  keg_only :versioned_formula
+
+  depends_on "pkg-config" => :build
+  depends_on "aom"
+  depends_on "aribb24"
+  depends_on "dav1d"
+  depends_on "fontconfig"
+  depends_on "freetype"
+  depends_on "frei0r"
+  depends_on "gnutls"
+  depends_on "lame"
+  depends_on "libass"
+  depends_on "libbluray"
+  depends_on "librist"
+  depends_on "libsoxr"
+  depends_on "libvidstab"
+  depends_on "libvmaf"
+  depends_on "libvorbis"
+  depends_on "libvpx"
+  depends_on "opencore-amr"
+  depends_on "openjpeg"
+  depends_on "opus"
+  depends_on "rav1e"
+  depends_on "rubberband"
+  depends_on "sdl2"
+  depends_on "snappy"
+  depends_on "speex"
+  depends_on "srt"
+  depends_on "svt-av1"
+  depends_on "tesseract"
+  depends_on "theora"
+  depends_on "webp"
+  depends_on "x264"
+  depends_on "x265"
+  depends_on "xvid"
+  depends_on "xz"
+  depends_on "zeromq"
+  depends_on "zimg"
+
+  uses_from_macos "bzip2"
+  uses_from_macos "libxml2"
+  uses_from_macos "zlib"
+
+  on_linux do
+    depends_on "alsa-lib"
+    depends_on "libxv"
+  end
+
+  on_intel do
+    depends_on "nasm" => :build
+  end
+
+  fails_with gcc: "5"
+
+  def install
+    args = %W[
+      --prefix=#{prefix}
+      --datadir=#{pkgshare}
```

Comment:
Without this `--datadir` override:

```
$ brew ls -v ffmpeg@5.1 | tail
/usr/local/Cellar/ffmpeg@5.1/5.1.3/share/ffmpeg/examples/demuxing_decoding.c
/usr/local/Cellar/ffmpeg@5.1/5.1.3/share/ffmpeg/examples/remuxing.c
/usr/local/Cellar/ffmpeg@5.1/5.1.3/share/ffmpeg/examples/metadata.c
/usr/local/Cellar/ffmpeg@5.1/5.1.3/share/ffmpeg/examples/http_multiclient.c
/usr/local/Cellar/ffmpeg@5.1/5.1.3/share/ffmpeg/libvpx-1080p.ffpreset
/usr/local/Cellar/ffmpeg@5.1/5.1.3/share/ffmpeg/libvpx-720p50_60.ffpreset
/usr/local/Cellar/ffmpeg@5.1/5.1.3/share/ffmpeg@5.1/python/convert_from_tensorflow.py
/usr/local/Cellar/ffmpeg@5.1/5.1.3/share/ffmpeg@5.1/python/convert.py
/usr/local/Cellar/ffmpeg@5.1/5.1.3/share/ffmpeg@5.1/python/convert_header.py
/usr/local/Cellar/ffmpeg@5.1/5.1.3/share/ffmpeg@5.1/python/tf_sess_config.py
```

Reply:
Actually, maybe we should install all these into `share/"ffmpeg"` instead.

Reply:
Makes me wonder if `pkgshare` really should return a directory with the version suffix.